package-build.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kctrl.carvel.dev/v1alpha1
+kind: PackageBuild
+metadata:
+  name: secretgen-controller.carvel.dev
+spec:
+  release:
+  - resource: {}
+  template:
+    spec:
+      app:
+        spec:
+          deploy:
+          - kapp: {}
+          template:
+          - ytt:
+              paths:
+              - config/package-bundle/config
```

Comment:
I think one interesting consequence of this is that if another team (perhaps someone that integrates and adds overlays to SGC such as TKG) previously relied on the path being `config` and they have stuff they can just add to that folder before template stage. This just changes that path to be more specific. I wonder what the consequences are.

Reply:
@rohitagg2020 your thoughts on this?

Reply:
@neil-hickey can you elaborate more on the scenario for which TKG currently apply overlays if you are aware of their flow?